vulcanai/__init__.py

```diff
@@ -14,3 +20,19 @@
                           defaults={'logfilename': log_output_path})
 
 logger = logging.getLogger(__name__)
+
+
+def set_global_seed(seed_value):
+    """
+    Sets all the random seeds.
+    :param seed_value: seed value
```

Comment:
Can you update this to the more verbose way we do function docs please?

Reply:
yeah, sorry, bad IDE defaults. done.